examples/fit_fake_data.py

```diff
@@ -42,7 +47,6 @@
 # }}}
 #}}}
 #{{{lmfitdata method
-M0,Mi,R1,vd = sp.symbols("M0 Mi R1 tau")
```

Comment:
again, no need to define the same thing twice

Reply:
I couldn't find the documentation in sympy for symbols to figure out what the real kwarg is but I know for lmfitdata it can't be in there but for fitdata it has to be in there...https://jmfrancklab.slack.com/archives/CV553TPRV/p1633962614001500